### DISCUSSION TITLE
La valeur de retour de App\entity\machin :: getTrucidFK() doit être une instance de App/entity/FamMachin, chaîne renvoyée

### DISCUSSION THREAD

**Discussion by AnthonyHaye** [symfony/symfony#48799] [Q&A] [answered]
Bonjour à tous
Je cherche à lier deux entités d'une base de donnée déjà existante, c'est dans le cadre d'une migration d'application php vers symfony.
Via une liaison ManyToOne, j'ai voulu récupérer la clé étrangère (nommé: pro_famid_fk), afin de récupérer certains champs de ce tableau.
Pouvez-vous m'orienter, je pense qu'il y a eu des discussions sur ce thème.

**Comment by Seb33300** [comment, marked as answer]
Bonjour,
As tu jeté un oeil à la documentation sur ce sujet ?
https://symfony.com/doc/current/doctrine/associations.html
